scenario_player/node_support.py

```diff
@@ -343,6 +347,15 @@ def _command(self):
                 else:
                     cmd.append(f"--{option_name}")
 
+        for option_name in self._options.keys() - seen:
+            option_value = self._options[option_name]
+            if isinstance(option_value, list):
+                cmd.extend([f"--{option_name}", *self._options[option_name]])
+            else:
+                cmd.extend([f"--{option_name}", self._options[option_name]])
```

Comment:
Flags will run in this case, too. So you will get both `--no-cap-mediation-fees True` and `--no-cap-mediation-fees ` as args.